code/game/objects/items/defib.dm

```diff
@@ -510,171 +517,202 @@
 
 	update_appearance()
 
-/obj/item/shockpaddles/proc/shock_pulling(dmg, mob/H)
-	if(isliving(H.pulledby)) //CLEAR!
-		var/mob/living/M = H.pulledby
-		if(M.electrocute_act(dmg, H))
-			M.visible_message(span_danger("[M] is electrocuted by [M.p_their()] contact with [H]!"))
-			M.emote("scream")
+/obj/item/shockpaddles/proc/shock_pulling(dmg, mob/target)
+	if(!isliving(target.pulledby)) //CLEAR!
+		return
+
+	var/mob/living/dumbass = target.pulledby
+	if(dumbass.electrocute_act(dmg, target))
+		dumbass.visible_message(span_danger("[dumbass] is electrocuted by [dumbass.p_their()] contact with [target]!"))
+		dumbass.emote("scream")
 
-/obj/item/shockpaddles/proc/do_disarm(mob/living/M, mob/living/user)
+/obj/item/shockpaddles/proc/do_disarm(mob/living/target, mob/living/user)
 	if(!DEFIB_CAN_HURT(src))
 		return
 	busy = TRUE
-	M.visible_message(span_danger("[user] touches [M] with [src]!"), \
-			span_userdanger("[user] touches [M] with [src]!"))
-	M.adjustStaminaLoss(60)
-	M.Knockdown(75)
-	M.set_jitter_if_lower(100 SECONDS)
-	M.apply_status_effect(/datum/status_effect/convulsing)
+	target.visible_message(span_danger("[user] touches [target] with [src]!"), \
+			span_userdanger("[user] touches [target] with [src]!"))
+	target.adjustStaminaLoss(60)
+	target.Knockdown(75)
+	target.set_jitter_if_lower(100 SECONDS)
+	target.apply_status_effect(/datum/status_effect/convulsing)
 	playsound(src,  'sound/machines/defib/defib_zap.ogg', 50, TRUE, -1)
-	if(HAS_TRAIT(M,MOB_ORGANIC))
-		M.emote("gasp")
-	log_combat(user, M, "zapped", src)
+	if(HAS_TRAIT(target, MOB_ORGANIC))
```

Comment:
This isn't even a trait, wtf

Reply:
lmao i just fixed up the names but didn't notice this bit, yeah i'll fix this